src/app/payments/send-lightning.ts

```diff
@@ -417,9 +429,9 @@ const executePaymentViaIntraledger = async <
           pubkey: recipientPubkey,
           paymentAmounts: paymentFlow,
 
-          amountDisplayCurrency: converter.fromUsdAmount(paymentFlow.usdPaymentAmount),
+          amountDisplayCurrency: amountDisplayCurrencyAsNumber,
           feeDisplayCurrency: 0 as DisplayCurrencyBaseAmount,
-          displayCurrency: DisplayCurrency.Usd,
+          displayCurrency: senderDisplayCurrency,
```

Comment:
same here, we need to pass `additionalCreditMetadata` (line 452)

Reply:
https://github.com/GaloyMoney/galoy/pull/2383#discussion_r1122315709